network-api/networkapi/templates/pages/base.html

```diff
@@ -16,15 +16,7 @@
     <link rel="icon" type="image/png" sizes="196x196" href="/_images/favicons/favicon-196x196@2x.png">
     <link rel="shortcut icon" href="/_images/favicons/favicon.ico">
     <title>{% block pageTitle %}Mozilla Foundation - {% if page.specifics.seo_title %}{{ page.specifics.seo_title }}{% else %}{{ page.title }}{% endif %}{% endblock %}</title>
-    <script type="application/ld+json">
-   {
-     "@context" : "http://schema.org",
-     "@type" : "Organization",
-     "legalName" : "Mozilla Foundation",
-     "url" : "https://foundation.mozilla.org",
-     "logo" : "https://foundation.mozilla.org/_images/mozlogo.jpg"
-    }
-   </script>
+    {% block orgSchema%}{% endblock %}
```

Comment:
We should decide on a naming convention and then make sure to update all of them. Python is a "no camelcase, ever" language, so that's probably worth following in Django templates, too. The choice between snake_case and kebab-case (omg those names) is a bit more arbitrary though, so I guess as long as we pick one, we can update everything to that.